### PR TITLE
replace WithChildren type with ReactNode

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard5**
## Summary

This PR removes the `WithChildren` type from all components that accept `children`, and replaces it for `ReactNode` by `React`. The children are no longer optional. The `WithChildren` file is removed.

### Governance

- [x] Documentation is added
- [x] Test cases are added or updated
- [x] I've read the contributing document https://github.com/minvws/.github/blob/master/CONTRIBUTING.md
- [x] I've read and understand the Code of Conduct https://github.com/minvws/.github/blob/master/CODE_OF_CONDUCT.md
- [x] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License
      https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/LICENSE.txt
      and the contributor license agreement https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/CLA.md

### CODE REVIEW

File: src/components/layout/index.tsx [hunk 1]
```diff
@@ -46,7 +45,9 @@ export function getLayout(layoutProps: LayoutProps, lastGenerated: string) {
 
 export default Layout;
 
-function Layout(props: WithChildren<LayoutProps & ILastGeneratedData>) {
+function Layout(
+  props: LayoutProps & ILastGeneratedData & { children: React.ReactNode }
```
Comment (VWSCoronaDashboard8) [inline src/components/layout/index.tsx:49]: Here you can integrate things with the already existing layout props

```suggestion
export interface LayoutProps extends ILastGeneratedData {
  url?: string;
  title: string;
  description?: string;
  openGraphImage?: string;
  twitterImage?: string;
  children: React.ReactNode;
}
```

File: src/components/legenda/index.tsx [hunk 2]
```diff
@@ -1,10 +1,14 @@
 import * as React from 'react';
-import { WithChildren } from '~/types/index';
+
+interface IProps {
```
Comment (VWSCoronaDashboard8) [inline src/components/legenda/index.tsx:3]: Nitpick; I would call it `LegendaProps` I'm not sure if we reached concensus yet over the `I`-prefix, but in most recent code I haven't seen it.

File: src/components/loadingPlaceholder/index.tsx [hunk 3]
```diff
@@ -1,7 +1,11 @@
-import { WithChildren } from '~/types/index';
 import styles from './loadingPlaceholder.module.scss';
 
-export function LoadingPlaceholder({ children }: WithChildren) {
+interface IProps {
```
Comment (VWSCoronaDashboard8) [inline src/components/loadingPlaceholder/index.tsx:3]: same comment

File: src/components/maxWidth/index.tsx [hunk 4]
```diff
@@ -1,6 +1,10 @@
-import { WithChildren } from '~/types/index';
 import styles from './maxWidth.module.scss';
 
-export function MaxWidth({ children }: WithChildren) {
+interface IProps {
```
Comment (VWSCoronaDashboard8) [inline src/components/maxWidth/index.tsx:3]: etc

File: src/components/scalingSVG/index.tsx [hunk 5]
```diff
@@ -1,14 +1,13 @@
 import CSS from 'csstype';
-import { WithChildren } from '~/types/index';
 import styles from './scalingSVG.module.scss';
 
 interface IProps {
   width: number;
   height: number;
-  children: WithChildren;
+  children: React.ReactNode;
```
Comment (VWSCoronaDashboard8) [inline src/components/scalingSVG/index.tsx:7]: 👍  this was a weird one

**Review by VWSCoronaDashboard8 — CHANGES_REQUESTED** [review]

File: src/components/legenda/index.tsx (hunk 2 above)
Comment (VWSCoronaDashboard4) [reply src/components/legenda/index.tsx:3]: I usually use the name of the component + `Props`; so `LegendaProps` in this case.

Comment (VWSCoronaDashboard1) [reply src/components/legenda/index.tsx:3]: Also, Legenda needs to be changed to Legend, since the former is a Dutch term. (My mistake, sorry)

**Pull request merged**